src/Grpc/test/testassets/InteropClient/InteropClient.cs

```diff
@@ -95,8 +95,8 @@ private InteropClient(ClientOptions options)
             var services = new ServiceCollection();
             services.AddLogging(configure =>
             {
-                configure.SetMinimumLevel(LogLevel.Trace);
```

Comment:
@JamesNK This is now failing with:

> F:\workspace\_work\1\s\src\Grpc\test\testassets\InteropClient\InteropClient.cs(103,35): error CS1061: 'SimpleConsoleFormatterOptions' does not contain a definition for 'DisableColors' and no accessible extension method 'DisableColors' accepting a first argument of type 'SimpleConsoleFormatterOptions' could be found (are you missing a using directive or an assembly reference?) [F:\workspace\_work\1\s\src\Grpc\test\testassets\InteropClient\InteropClient.csproj]

Should I not merge this forward from preview8? Did I resolve the conflicts wrong?

Reply:
Keep the changes in release/5.0 for the gRPC interop tests. They are working correctly there